releases/4.0.3.md

```diff
@@ -0,0 +1,153 @@
+---
+title: TiDB v4.0.3 Release Notes
+category: Releases
+aliases: ['/docs/dev/releases/v4.0.3/']
+---
+
+# TiDB v4.0.3 Release Notes
+
+Release date: July 16, 2020
+
+TiDB version: v4.0.3
+
+## New Features
+
```

Comment:
+ TiDB Dashboard
    - Display detailed TiDB Dashobard version information [#679](https://github.com/pingcap-incubator/tidb-dashboard/pull/679)
    - Show browser compatibility notice for unsupported browsers or outdated browsers [#654](https://github.com/pingcap-incubator/tidb-dashboard/pull/654)
    - Support searching in statement page [#658](https://github.com/pingcap-incubator/tidb-dashboard/pull/658)